spec.html

```diff
@@ -1024,6 +1024,7 @@ <h1>Mathematical Operations</h1>
       <emu-note>
         <p><emu-eqn>floor(_x_) = _x_ - (_x_ modulo 1)</emu-eqn>.</p>
       </emu-note>
+      <p>An <dfn id="interval">interval</dfn> from lower bound _a_ to upper bound _b_ is a possibly-infinite, possibly-empty set of numeric values of the same numeric type. A numeric value _x_ is included in the interval if it satisfies the criteria _a_ &lt; _x_ and _x_ &lt; _b_. If the lower bound is described as inclusive, _a_ &le; _x_ is used instead of _a_ &lt; _x_. If the upper bound is described as inclusive, _x_ &le; _b_ is used instead of _x_ &lt; _b_. An <dfn id="inclusive-interval">inclusive interval</dfn> is an interval where both bounds are inclusive. *NaN* is never included in an interval. An interval with an exclusive bound of *+0*<sub>𝔽</sub> or *-0*<sub>𝔽</sub> includes neither *+0*<sub>𝔽</sub> nor *-0*<sub>𝔽</sub>. An interval with an inclusive bound of *+0*<sub>𝔽</sub> or *-0*<sub>𝔽</sub> includes both *+0*<sub>𝔽</sub> and *-0*<sub>𝔽</sub>.</p>
```

Comment:
Also

> `An interval with an inclusive bound of *+0*<sub>𝔽</sub> or *-0*<sub>𝔽</sub> includes both *+0*<sub>𝔽</sub> and *-0*<sub>𝔽</sub>.`

This is not consistent with how we actually use the terms. `ToUint16`, for example, never produces -0.

Reply:
It is true that `ToUint16` does not produce `-0` but the previous phrasing didn't guarantee that either. We can add an additional "this operation never produces `-0` or something to those sections.

Reply:
Alternatively, we could significantly complicate things and "order" the zeroes using the rules

* an inclusive lower bound of `-0` includes both zeroes
* an inclusive lower bound of `+0` includes `+0` but not `-0`
* an exclusive lower bound of `-0` includes `+0` but not `-0`
* an exclusive lower bound of `+0` includes neither zero
* an inclusive upper bound of `-0` includes `-0` but not `+0`
* an inclusive upper bound of `+0` includes both zeroes
* an exclusive upper bound of `-0` includes neither zero
* an exclusive upper bound of `+0` includes `-0` but not `+0`

Reply:
Or just say "for the purposes of defining intervals, -0 is considered to be strictly less than +0".